FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs

```diff
@@ -1161,6 +1164,236 @@ public async Task<FirebaseToken> VerifySessionCookieAsync(string sessionCookie)
             return decodedToken;
         }
 
+        /// <summary>
+        /// Looks up an OIDC auth provider configuration by the provided ID.
+        /// </summary>
+        /// <returns>A task that completes with a <see cref="OidcProviderConfig"/>.</returns>
+        /// <exception cref="ArgumentException">If the provider ID is null, empty or does not
+        /// contain the <c>oidc.</c> prefix.</exception>
+        /// <exception cref="FirebaseAuthException">If the specified provider config does not
+        /// exist.</exception>
+        /// <param name="providerId">The provider ID corresponding to the OIDC provider config
+        /// to return.</param>
+        public async Task<OidcProviderConfig> GetOidcProviderConfigAsync(string providerId)
+        {
+            return await this.GetOidcProviderConfigAsync(providerId, default(CancellationToken))
+                .ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Looks up an OIDC auth provider configuration by the provided ID.
+        /// </summary>
+        /// <returns>A task that completes with a <see cref="OidcProviderConfig"/>.</returns>
+        /// <exception cref="ArgumentException">If the provider ID is null, empty or does not
+        /// contain the <c>oidc.</c> prefix.</exception>
+        /// <exception cref="FirebaseAuthException">If the specified provider config does not
+        /// exist.</exception>
+        /// <param name="providerId">The provider ID corresponding to the OIDC provider config
+        /// to return.</param>
+        /// <param name="cancellationToken">A cancellation token to monitor the asynchronous
+        /// operation.</param>
+        public async Task<OidcProviderConfig> GetOidcProviderConfigAsync(
+            string providerId, CancellationToken cancellationToken)
+        {
+            var providerConfigManager = this.IfNotDeleted(
+                () => this.providerConfigManager.Value);
+            return await providerConfigManager
+                .GetOidcProviderConfigAsync(providerId, cancellationToken)
+                .ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Looks up a SAML auth provider configuration by the provided ID.
+        /// </summary>
+        /// <returns>A task that completes with a <see cref="SamlProviderConfig"/>.</returns>
+        /// <exception cref="ArgumentException">If the provider ID is null, empty or does not
+        /// contain the <c>saml.</c> prefix.</exception>
+        /// <exception cref="FirebaseAuthException">If the specified provider config does not
+        /// exist.</exception>
+        /// <param name="providerId">The provider ID corresponding to the SAML provider config
```

Comment:
The ID of the SAML provider config to return.

Reply:
Done